.eslintrc

```diff
@@ -14,7 +14,8 @@
     "consistent-return": 0,
     "comma-dangle": 0,
     "spaced-comment": 0,
-    "global-require": 0
+    "global-require": 0,
+    "no-param-reassign": ["error", {"props": false}]
```

Comment:
@aysbg sure haha. What are you stealing it for? :). And how did you come by this PR haha? Or is there some connection I don't know about? :P